ui/panels-plugin/src/plugins/gauge-chart/thresholds.ts

```diff
@@ -11,20 +11,73 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export const ThresholdColorsMap = {
-  green: 'rgb(115, 191, 105)',
-  orange: 'rgba(237, 129, 40, 0.89)',
-  red: 'rgba(245, 54, 54, 0.9)',
+import { merge } from 'lodash-es';
+
+export const ThresholdColors = {
+  GREEN: 'rgba(115, 191, 105, 1)',
+  ORANGE: 'rgba(237, 129, 40, 0.9)',
+  RED: 'rgba(245, 54, 54, 0.9)',
```

Comment:
does these colors will change depending of the theme ?

Reply:
Not currently, this ThresholdColors implementation is temporary--I added a TODO item for a future PR when I will look into extending MUI themes to work with ECharts themes ([1](https://echarts.apache.org/en/api.html#echarts.registerTheme), [2](https://echarts.apache.org/en/theme-builder.html))